frontend/common/src/components/Link/ScrollToLink.tsx

```diff
@@ -18,6 +19,7 @@ const ScrollToLink = ({ to, children, ...rest }: ScrollToLinkProps) => {
   const handleClick = (e: React.MouseEvent<HTMLAnchorElement>) => {
     if (targetSection) {
       e.preventDefault();
+      HISTORY.push(`#${to}`);
       targetSection.scrollIntoView({
         behavior: "smooth",
         block: "start",
```

Comment:
Any reason you removed smooth scroll in the [router](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4216/commits/1fdaa7241021f72d0c372337db47abfde8185bfd) but not here?

Reply:
Smooth scroll is nice when you are clicking a link but looked kinda janky when it was happening on load. Open to changing if others prefer it on load.